config/default.yml

```diff
@@ -3569,11 +3569,12 @@ Style/RaiseArgs:
   StyleGuide: '#exception-class-messages'
   Enabled: true
   VersionAdded: '0.14'
-  VersionChanged: '0.40'
+  VersionChanged: '0.87'
   EnforcedStyle: exploded
   SupportedStyles:
     - compact # raise Exception.new(msg)
     - exploded # raise Exception, msg
+  AllowedNonExplodedTypes: []
```

Comment:
Probably we should stick to the "compact" terminology here, if we decide to go forward with this change.

Reply:
I can get behind that, thanks. I was trying to convey that it only affects the "Exploded" style. But I also agree with your comment that it's awkward that this option only affects one of the styles.